doc/source/reference/routines.rst

```diff
@@ -24,6 +24,7 @@ indentation.
    routines.ctypeslib
    routines.datetime
    routines.dtype
+   routines.dtypes
```

Comment:
@ngoldbaum this file doesn't actually exist. Did you forget to commit it, or is this some leftover?

Reply:
Hmm, that's odd, must have forgotten to `git add` it back when I made this PR. Will push a fix tomorrow.